doc/user_guide/docker/docker_usage.md

```diff
@@ -142,10 +142,35 @@ port to the same port then you can connect with http://localhost:49494.
 └─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
 
 The default password is "ai-lab".
-To update the password as user root run
+To update the password login to the Docker container as user root and run
     /root/jupyterenv/bin/jupyter-lab server password
 ```
 
 Using an internet browser you then can connect to the Jupyter server running in the Docker container in order to follow the tutorials presented by a set of Jupyter notebooks, see [Connecting to Jupyter Service](../user_guide.md#connecting-to-jupyter-service).
 
 For parameter `<host>`: If your daemon machine is identical to the machine your browser is running on then you can replace `<host>` by `localhost` otherwise please use the IP address of the daemon machine.
+
+The following section explains how to login to the Docker container in order to change the default password for example.
+
+## Login to the Docker container
+
+To update the password you first need to login to the Docker container.
```

Comment:
I would rephrase it a bit. The way it's written it may seem like you should first login to the Docker container and then run the below command.

To update the password you should log in to the Docker container.

First, you need to find out the container's ID.

The following command shows ...